yml/dp02_patches.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: dp02_dc2_images
+"@id": "#dp02_dc2_images"
+description: Information about image data products in DP0.2 processing of DC2
+tables:
+- name: lsst_skymap
+  "@id": "#lsst_skymap"
+  description: Static information about the tract-and-patch skymap used for coadds
+  columns:
+  - name: lsst_tract
+    "@id": "#lsst_skymap.lsst_tract"
+    description: "ID number of the top level, 'tract', of the standard LSST skymap"
+    nullable: false
+    ivoa:ucd: meta.id.part
+    tap:std: 0
+    tap:principal: 1
+    ivoa:unit:
+    datatype: int
+  - name: lsst_patch
+    "@id": "#lsst_skymap.lsst_patch"
+    description: "ID number of the second level, 'patch', of the standard LSST skymap"
+    nullable: false
+    ivoa:ucd: meta.id.part
+    tap:std: 0
+    tap:principal: 1
+    ivoa:unit:
+    datatype: int
```

Comment:
While I believe that a patch does have a representation as a single int, the canonical patch index is a tuple of x,y coordinates: https://pipelines.lsst.io/py-api/lsst.skymap.PatchInfo.html#lsst.skymap.PatchInfo.getIndex

Reply:
FWIW, all other instances of "patch" in the catalog schemas currently seem to be the single int (long) version, except the upcoming alien DC2 `TruthSummary` table...

Reply:
Perhaps we want both forms in each row of this table -- then it could also be `JOIN`ed across to translate between the two forms in database queries?